### PR TITLE
Copy lesson over

### PR DESCRIPTION
**Pull request by kcleland0818**

### CODE REVIEW

**Review by developerjaym — APPROVED** [review]

**Review by linda-seiter — APPROVED** [review]
Can we show example input/output examples of an exception begin thrown or not?
Have them debug the program to see the exception control flow?

File: README.md [hunk 1]
```diff
@@ -2,31 +2,115 @@
...
+    Scanner inputScanner = new Scanner(System.in);
+    int userNumber = inputScanner.nextInt();
+    System.out.println("The user entered " + userNumber);
+}
+```
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+The `nextInt()` method of the `Scanner` class can throw a
```
Comment (developerjaym) [inline README.md]: Consider changing 'a' to 'an'

File: README.md [hunk 2]
```diff
@@ -2,31 +2,115 @@
...
-- [Resource Link 1](example.com)
-- [Resource Link 2](example.com)
+A **try-catch block** tells the compiler that the code anticipates the
+unexpected exceptions and that it is prepared to handle it. A `try` block can
+have as many statements in it as we want and is enclosed with curly braces. Any
+exceptions that are thrown by any code inside the `try` block will:
+
+- Cause the execution of the statements inside the `try` block to be interrupted
+  immediately.
+- Cause the execution to jump straight pass the remaining statements (if any)
```
Comment (developerjaym) [inline README.md]: I think 'pass' should be 'past'

**Review by developerjaym — APPROVED** [review]

**Review by linda-seiter — APPROVED** [review]

**Review by developerjaym — APPROVED** [review]

**Comment by developerjaym** [comment]
Is there a lesson somewhere explaining how and why to throw an exception?

**Comment by kcleland0818** [comment]
> Is there a lesson somewhere explaining how and why to throw an exception?

Linda suggested actually talking more in-depth about exceptions when addressing inheritance in a later module.

**Pull request merged**